CWE/decisions/Romania.txt

```diff
@@ -29,4 +30,195 @@ political_decisions = {
 			factor = 1
 		}
 	}
-}
\ No newline at end of file
+
+	oil_modernization_completion = {
+		picture = "rom_oil"
+		potential = {
+			AND = {
+				owns = 667
+				owns = 672
+				owns = 665
+			}
+			OR = {
+				has_country_flag = nicolae_ceausescu_flag
+				has_country_flag = rom_manescu
+				has_country_flag = gheorghe_apostol_romania
+				has_country_flag = rom_nicu
+				has_country_flag = rom_zoia
```

Comment:
This decision should be set as fired in romania's country history file (under the 1992 section) using: 

decision = oil_modernization_completion

Reply:
This would be like the other one, right? I don't need it fired in the 1992 start, and it shouldn't be accessible from the 1992 start.

Reply:
Alright